src/preact-render-spy.js

```diff
@@ -134,17 +134,49 @@ class SpyWrapper {
     this.fragment = document.createDocumentFragment();
   }
 
-  find(selector) {
-    return new FindWrapper(this, [this.vdomMap.get('root')], selector);
-  }
-
   render(vdom) {
     this.component = render(
       spyWalk(this, setVDom(this, 'root', vdom), 0),
       this.fragment
     );
     return this;
   }
+
+  find(selector) {
```

Comment:
If we were to set `this.context = this`, `this.length = 1` and `this[0] = this.vdomMap.get('root')` every time we render, we can just make `RenderContext` extend `FindWrapper` right?